numpy/ma/tests/test_core.py

```diff
@@ -556,23 +556,23 @@ def test_str_repr(self):
         a[1,1] = np.ma.masked
         assert_equal(
             repr(a),
-            textwrap.dedent('''\
+            textwrap.dedent(f'''\
             masked_array(
               data=[[1, 2, 3],
                     [4, --, 6]],
               mask=[[False, False, False],
                     [False,  True, False]],
-              fill_value=999999,
+              fill_value={np.array(999999)[()]!r},
```

Comment:
Why not just put `np.float64(999999.0)` here (and below)? Really would hope we're not changing it again!

Reply:
Because this is an integer and this is needed because it's either int64 or int32

Reply:
Ah, darn.